ballerina/Package.md

```diff
@@ -3,7 +3,7 @@
 The I/O library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
 </a> language.
 
-This package provides APIs for file input/output operations and the console input/output.
+This package provides APIs for file input/output operations, and the console input/output.
```

Comment:
```suggestion
This package provides APIs for file input/output operations and the console input/output.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/477/commits/b940cf5754398095dd38803252c05359a666001c